Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -15,10 +15,27 @@
 
 @implementation BPPacker
 
++ (NSMutableSet *)fullyExpandedTests:(NSArray *)configTestCases fromTestFile:(BPXCTestFile *)xctFile {
```

Comment:
`expandTests: withTestFile:`, also if this is a helper method, move to the bottom and add

`#pragma mark - private helpers`

before the method